docker-compose-installed.yaml

```diff
@@ -115,15 +123,20 @@ services:
       context: .
       dockerfile: ./rpc/Dockerfile-installed
       args:
-        ISOLATION_ID: ${ISOLATION_ID}
+        - ISOLATION_ID=${ISOLATION_ID}
+        - http_proxy
+        - https_proxy
+        - no_proxy
     image: sawtooth-seth-rpc:${ISOLATION_ID}
     container_name: seth-rpc
     volumes:
       - ./build/debs:/build
     depends_on:
       - validator
+    expose:
+      - 3030    
     ports:
-      - 3030:3030
+      - '3030:3030'
```

Comment:
Was this causing any issues? Though it is a best practice to configure ports as a string in docker-compose, this should be a different commit and PR if it does not relate to the bug being fixed here.

Reply:
Issue I'm solving is port 3030 connection refused issue on Seth. AFAIK Hyperledger/sawtooth-seth will not standup with current stable docker 18:03.1-ce without yaml files changes on Mac 10.13.5 or Ubuntu lts 16.04 to 18.04.